Podfile

```diff
@@ -18,7 +18,7 @@ use_frameworks!
 # Warning: our internal tooling depends on the name of this variable name, so be sure not to change it
 $matrixSDKVersion = '= 0.23.10'
 # $matrixSDKVersion = :local
-# $matrixSDKVersion = { :branch => 'dinum_v0.20.15'}
+# $matrixSDKVersion = { :branch => 'develop'}
```

Comment:
Can we use yet the SDK on develop branch for Tchap ?

Reply:
The SDK is now the same for Tchap and Element.
So it will not be an issue to have `develop` and I think `dinum_v0.20.15` branch has been deleted (or won't be used anymore).